### DISCUSSION TITLE
Entity "homeassistant/number" is unknown

### DISCUSSION THREAD

**Discussion by groggy99** [Koenkk/zigbee2mqtt#22688] [General]
Hi there,

can someone tell me what this error is about?

[2024-05-18 20:10:40] error:    z2m: Entity 'homeassistant/number' is unknown

He appears directly after starting z2m. I think since z2m 1.36 or 1.37. 
On version 1.34 it does not appear.

**Comment by francisp2** [comment]
What version of HA are you using ?

**Reply by groggy99** [reply]
Hi, it is 2024.4.4.
